app/views/users/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <% presenter = Users::EditPresenter.new(self) %>
 <% content_for(:head) do %>
-  <title><%= presenter.page_title %> | t("views.users.delete_account.edit_user")</title>
```

Comment:
`users.delete_account.edit_user` key should be removed if it's not used anywhere else